superset/utils/machine_auth.py

```diff
@@ -143,6 +144,31 @@ def get_auth_cookies(user: User) -> dict[str, str]:
 
         return cookies
 
+    @staticmethod
+    def get_auth_cookie_and_csrf_token(user: User) -> tuple[dict[str, str], str]:
```

Comment:
We could check for `app.config["WTF_CSRF_ENABLED"]` here and DRY the code a bit more by removing `get_auth_cookies`